sphinx/src/orga/00_intro.rst

```diff
@@ -1,12 +1,7 @@
-Organisation Documentation
-##########################
+# Organization
```

Comment:
This change is not needed, is it?

Reply:
You mean the # ? or the word documentation?

Reply:
oh wait, I got it :)